### ISSUE TITLE
install help on MacBook Pro

### ISSUE THREAD

**Issue by SL33p3rawakes** [AllYarnsAreBeautiful/ayab-desktop#263]
Hi there.
My first attempt to download and install AYAB looked exactly this [https://github.com/AllYarnsAreBeautiful/ayab-desktop/issues/241](url)
I'm running on a MacBook Pro with High Sierra (10.13.4) and had same "installing python" screen and everything.  Because the installation was taking too long (I thought) and after several reboots and launching AYAB again. I moved AYAB to Trash and thought I'd start again.  I think I was on the correct path, but I've made things worse.
So after the 2nd download of AYAB (in Applications folder), when I click it to launch (for first initial attempt) a little window pops up and disappears almost instantly.  So based up on that previous thread, here are my findings:
-can't find the /tmp/ayab_install.log
-can't find the /Users/VJ/.pyenv folder
-I get this following from this ($ xcode-select --install)
<img width="572" alt="screen shot 2018-05-16 at 6 47 17 am" src="https://user-images.githubusercontent.com/39334579/40113490-9ee66a50-58d7-11e8-9d88-b80d3d5d759b.png">

Any help would be most appreciated.  I apologize if this is posted in the wrong forum or thread.

**Comment by SL33p3rawakes** [comment]
Update:
Here is the install log.
<img width="632" alt="screen shot 2018-05-16 at 7 34 53 am" src="https://user-images.githubusercontent.com/39334579/40115138-6d727396-58dd-11e8-98b4-75b808b964a0.png">
and I found the .pyenv directory, but it won't let me remove/delete it.
<img width="574" alt="screen shot 2018-05-16 at 7 37 58 am" src="https://user-images.githubusercontent.com/39334579/40115149-7be64c54-58dd-11e8-9e4c-fe40ecde14df.png">

**Comment by SL33p3rawakes** [comment]
was able to delete the .pyenv directory and restarted AYAB...fingers crossed.
<img width="509" alt="screen shot 2018-05-16 at 8 01 01 am" src="https://user-images.githubusercontent.com/39334579/40115776-87a59598-58df-11e8-82d0-ae672dc960a5.png">

**Comment by SL33p3rawakes** [comment]
It worked!  AYAB opened without issue!

**Comment by SL33p3rawakes** [comment]
This window is still open and spinning at the bottom.  Will it eventually close?  
<img width="506" alt="screen shot 2018-05-16 at 8 37 51 am" src="https://user-images.githubusercontent.com/39334579/40117294-89339c34-58e4-11e8-8406-96e392f47f60.png">

**Comment by oskay** [comment]
I'm glad to hear that you got it working-- did you need to change anything besides removing the pyenv directory?

That launcher window will remain open when AYAB runs. The plan is to eventually hide it in a future version.

**Comment by SL33p3rawakes** [comment]
All I did (once I figured out HOW to remove it) was remove that pyenv directory.  Clicked the AYAB launcher and away we went.  Ok, I wasn't sure if that window runs in the background while AYAB is active and open or not.    Glad to know it is 'normal.'
Once I get the machine connected to the laptop I'll know for sure if all is well, until then at least progress is being made.  I'll post my success or beg for help once I get to that point - should be within the next week or so.

thanks for listening (reading)....
